LICENSE

```diff
@@ -107,6 +107,18 @@ The externally maintained libraries used by Node.js are:
     THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   """
 
+- ittapi, located at deps/v8/third_party/ittapi, is licensed as follows:
```

Comment:
There has to be a change to [`tools/license-builder.sh`](https://github.com/nodejs/node/blob/main/tools/license-builder.sh) for this, otherwise it will be reverted next time we run the script.

Reply:
Hi targos, are you suggesting that I need to modify license-builder.sh to add a new line of addLicense to generate this ittapi license content?

Reply:
@lucshi Yes!